tck/features/clauses/create/Create1.feature

```diff
@@ -0,0 +1,137 @@
+#
+# Copyright (c) 2015-2019 "Neo Technology,"
+# Network Engine for Objects in Lund AB [http://neotechnology.com]
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Attribution Notice under the terms of the Apache License 2.0
+#
+# This work was created by the collective efforts of the openCypher community.
+# Without limiting the terms of Section 6, any Derivative Work that is not
+# approved by the public consensus process of the openCypher Implementers Group
+# should not be described as “Cypher” (and Cypher® is a registered trademark of
+# Neo4j Inc.) or as "openCypher". Extensions by implementers or prototypes or
+# proposals for change that have been documented or implemented should only be
+# described as "implementation extensions to Cypher" or as "proposed changes to
+# Cypher that are not yet approved by the openCypher community".
+#
+
+#encoding: utf-8
+
+Feature: Create1 - Creating nodes
+
+  Scenario: [1] Creating a single node
+    Given any graph
+    When executing query:
+      """
+      CREATE ()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes | 1 |
+
+  Scenario: [2] Creating two nodes
+    Given any graph
+    When executing query:
+      """
+      CREATE (), ()
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes | 2 |
+
+  Scenario: [3] Creating a single node with a label
+    Given an empty graph
+    When executing query:
+      """
+      CREATE (:Label)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes  | 1 |
+      | +labels | 1 |
+
+  Scenario: [4] Creating a single node with multiple labels
+    Given an empty graph
+    When executing query:
+      """
+      CREATE (:A:B:C:D)
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes  | 1 |
+      | +labels | 4 |
+
+  Scenario: [5] Creating a single node with a property
+    Given any graph
+    When executing query:
+      """
+      CREATE ({created: true})
+      """
+    Then the result should be empty
+    And the side effects should be:
+      | +nodes      | 1 |
+      | +properties | 1 |
+
+  Scenario: [6] Creating a single node with a property and returning it
```

Comment:
This is in the purview of RETURN as it stands. I suggest moving it to uncategorized for now, OR adding this as a control query (like in some scenarios in the other CreateX files), and attach it to Scenario 5 above

Reply:
In none of theses cases, a control query will do the same thing, because here CREATE not only creates the node, but also bind the node. So the question is, if the new variable correctly exposes the properties. What is checked is that RETURN (or any other subsequent clause) sees the properties of the created elements without intermediate matching. 

It could be discussed, if the correct binding that should be checked as a RETURN scenario. In the MATCH case, I have argued in that direction. Here, I not so sure. Maybe it is interoperability between CREATE and RETURN. 

Further, I am reluctant to modify the scenarios. We should try to categorize without modifying them. Organize the content first. Improve the content later.

Reply:
I think it is indeed either RETURN or an interoperability case (I lean more towards the latter). But fair enough, as this is simply a first pass-through, this is nit-picking - so let's leave for later as you suggest